javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/Instrumenter.java

```diff
@@ -237,7 +237,7 @@ protected ReferenceMatcher getInstrumentationMuzzle() {
     }
 
     protected boolean defaultEnabled() {
-      return Config.get().isIntegrationsEnabled();
+      return Config.get().getBooleanProperty("otel.integrations.enabled", true);
```

Comment:
This should be cached, I think.

Reply:
It probably should - but its value is modified in the unit test heavily, so I've decided to just leave it here. This method is only called during the instrumenter initialization, so calling the `Config` here adds no overhead once the agent starts.